packages/common/src/ui/components/navigation/Tabs/TabKeybindings.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
```

Comment:
lifted this out to a component.. because it was fun to do 🤷 
It does make it easier to keep them consistent, and is slightly easier to add new tabs in to the key bindings